src/components/Card/OverviewCard/index.tsx

```diff
@@ -1,12 +1,16 @@
-import { ReactNode } from 'react'
+/* eslint-disable object-curly-newline */
```

Comment:
I'd like to disable it globally or set it `consistent": true`. Just my preference. It depends on you to update or keep it in this PR.

Ref: https://eslint.org/docs/latest/rules/object-curly-newline

Reply:
I think this rule can be removed because similar constraints can be applied prettier.